### ISSUE TITLE
Information about VSTS version bump

### ISSUE THREAD

**Issue by marceloavf** [BMuuN/vsts-assemblyinfo-task#16]
How long it takes to the version of vsts-assemblyinfo-task bump to the latest inside VSTS ?

Thanks!

**Comment by BMuuN** [comment]
I'll publish a new version later this evening which will include the SemVer fix.  You should see the change instantly.

I'll report back here once a new version has been published.

**Comment by BMuuN** [comment]
Published version 1.0.29 of the plugin, let me know if it addresses your requirements.

To ensure you have the latest version look for version 1.0.39 of the task in your build output:
```
==============================================================================
Task         : Assembly Info
Description  : Set assembly information such as Version, Copyright and Trademark etc.
Version      : 1.0.39
Author       : Bleddyn Richards
Help         : [More Information](https://marketplace.visualstudio.com/items?itemName=bleddynrichards.Assembly-Info-Task)
==============================================================================
```

**Comment by marceloavf** [comment]
Awesome, works great!
Thanks.
